Functions.Templates/Templates/TimerTrigger-CSharp-Isolated/TimerTriggerCSharp.cs

```diff
@@ -11,7 +9,7 @@ public static class TimerTriggerCSharp
         [FunctionName("TimerTriggerCSharp")]
```

Comment:
Function?

Reply:
Oops! Missed this ! Good catch.

Reply:
Thank you thank you! Looks like I missed here and in Service Bus.